astroid/rebuilder.py

```diff
@@ -1382,6 +1382,20 @@ def visit_global(self, node: "ast.Global", parent: NodeNG) -> nodes.Global:
                 self._global_names[-1].setdefault(name, []).append(newnode)
         return newnode
 
+    def _find_else_keyword(self, node: "ast.If") -> Tuple[Optional[int], Optional[int]]:
+        """Get the line number and column offset of the `else` keyword."""
+        if not self._data or not node.orelse:
+            return None, None
+
+        end_lineno = node.orelse[0].lineno - 1
+
+        # pylint: disable-next=unsubscriptable-object
```

Comment:
Is there an issue in pylint for that false positive ?

Reply:
Issue in `pylint` I think. Marc added disables as well in his PRs.